using-kubecost/carbon-costs.md

```diff
@@ -0,0 +1,21 @@
+# Carbon Costs
+
+Carbon Costs is a cost metric added to both [Allocation](/using-kubecost/navigating-the-kubecost-ui/cost-allocation/README.md) and [Assets](/using-kubecost/navigating-the-kubecost-ui/assets.md) dashboards. Carbon Costs are measured in KG CO2e, which is defined by the EPA as:
+
+> Carbon dioxide equivalent or CO2e means the number of metric tons of CO2 emissions with the same global warming potential as one metric ton of another greenhouse gas, and is calculated using Equation A-1 in 40 CFR Part 98.
+
+For more information on how carbon costs are calculated, visit the [Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org/) website.

```

Comment:
@kaelanspatel @bstuder99 what about a 1-2 line summary about how these costs are calculated/estimated?

Reply:
```suggestion
Kubecost uses carbon coefficient data generated by the [Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org/) to associate estimated carbon costs to Disk, Node, and Network Assets based on their runtime. This carbon cost is then distributed to Allocations. For more information on how these coefficients are calculated, visit the Cloud Carbon Footprint's [methodology page](https://www.cloudcarbonfootprint.org/docs/methodology).
```
Definitely. The coefficients themselves come from the CCF.